test-config/wdio.browserstack.conf.js

```diff
@@ -32,7 +36,6 @@ const allCapabilities = [
 ]
 
 exports.config = {
-  sync: true,
```

Comment:
doesn't seem to work correctly in all cases and is deprecated in version 7 anyway